targets/TARGET_STM/sleep.c

```diff
@@ -84,6 +85,10 @@ void hal_deepsleep(void)
     // After wake-up from STOP reconfigure the PLL
     SetSysClock();
 
+    TIM_HandleTypeDef TimMasterHandle;
+    TimMasterHandle.Instance = TIM_MST;
+    __HAL_TIM_SET_COUNTER(&TimMasterHandle, EnterTimeUS);
```

Comment:
Note: This is one of the things we should cover in the API docs, as deepsleep in many cases would stop us ticker clocks/resets, thus should it resume time or not (currently user's assumption must be made).